arcade/paths.py

```diff
@@ -169,13 +239,15 @@ class AStarBarrierList:
     Class that manages a list of barriers that can be encountered during
     A* path finding.
 
-    :param Sprite moving_sprite: Sprite that will be moving
-    :param SpriteList blocking_sprites: Sprites that can block movement
-    :param int grid_size: Size of the grid, in pixels
-    :param int left: Left border of playing field
-    :param int right: Right border of playing field
-    :param int bottom: Bottom of playing field
-    :param int top: Top of playing field
+    Attributes:
+        moving_sprite (BasicSprite): Sprite that will be moving
+        blocking_sprites (SpriteList): Sprites that can block movement
+        grid_size (int): Size of the grid, in pixels
+        left (int): Left border of playing field
+        right (int): Right border of playing field
+        bottom (int): Bottom of playing field
+        top (int): Top of playing field
+        barrier_list (Optional[Set]): SpriteList of barriers to use in _AStarSearch, None if not recalculated
```

Comment:
As mentioned earlier, I am not in favor of switching to Google doc string style in this PR.